python/PiFinder/calc_utils.py

```diff
@@ -53,3 +53,12 @@ def radec_to_altaz(self, ra, dec, alt_only=False):
         else:
             az = 360 - _az
         return alt, az
+
+
+import json
```

Comment:
I'd like to keep all module level imports up top.  After merging I'll re-arrange these :-)